src/wireviz/Harness.py

```diff
@@ -344,6 +345,44 @@ def create_graph(self) -> Graph:
             dot.node(cable.name, label=f'<\n{html}\n>', shape='box',
                      style=style, fillcolor=translate_color(bgcolor, "HEX"))
 
+        def typecheck(name: str, var, type) -> None:
+            if not isinstance(var, type):
+                raise Exception(f'Unexpected value type of {name}: {var}')
+
+        # TODO?: Differ between override attributes and HTML?
+        if self.tweak.override is not None:
+            typecheck('tweak.override', self.tweak.override, dict)
+            for k, d in self.tweak.override.items():
+                typecheck(f'tweak.override.{k} key', k, str)
+                typecheck(f'tweak.override.{k} value', d, dict)
+                for a, v in d.items():
+                    typecheck(f'tweak.override.{k}.{a} key', a, str)
+                    typecheck(f'tweak.override.{k}.{a} value', v, str)
+
+            # Override generated attributes of selected entries matching tweak.override.
+            for i, entry in enumerate(dot.body):
+                if isinstance(entry, str):
+                    # Find a possibly quoted keyword after leading TAB(s) and followed by [ ].
+                    match = re.match(r'^\t*(")?((?(1)[^"]|[^ "])+)(?(1)") \[.*\]$', entry, re.S)
+                    keyword = match and match[2]
+                    if keyword in self.tweak.override.keys():
+                        for attr, value in self.tweak.override[keyword].items():
+                            if len(value) == 0 or ' ' in value:
+                                value = value.replace('"', r'\"')
+                                value = f'"{value}"'
+                            # TODO?: If value is None: delete attr, and if attr not found: append it?
```

Comment:
both ideas make sense

Reply:
Yes, that was the plan. The current implementation should be enough to solve the original issue and a lot of similar experimental tweaking, but I don't expect a huge number of users will need such tweaking in their harnesses. Maybe it'll mainly be a tool for developers to test variations of `.gv` output before suggesting a proper implementation of a new feature.

Reply:
I just implemented these features as separate commits in this PR as well.